manifests/klusterlet/managed/klusterlet-work-clusterrole-execution.yaml

```diff
@@ -20,3 +20,7 @@ rules:
 - apiGroups: ["rbac.authorization.k8s.io"]
   resources: ["clusterroles", "roles"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete", "escalate", "bind"]
+# Allow OCM addons to setup metrics collection with Prometheus
+- apiGroups: ["monitoring.coreos.com"]
+  resources: ["servicemonitors"]
```

Comment:
`servicemonitors` is a namespace scoped CRD, as we discussed before in [Code refactor to split permission for registration/work](https://docs.google.com/document/d/1SWT4jcBhn6c3dPt2TyNdh4NqfdxSSDiOlSSYgy6QgqM/edit#heading=h.ba0yojrlee5r), this namespace scoped resource should be put this into `Role open-cluster-management:{{ .KlusterletName }}-work:execution` (this Role is not created so far). @qiujian16 any comments for this?

Reply:
I think this is for addon to create their service monitors? It is  ok to put here if we do not have that role yet, but yes we should put it in a role in the future. A TODO here would be helpful.

Reply:
That's right. It's for an addon to create a service monitor.

I just added the `TODO` comment as requested.